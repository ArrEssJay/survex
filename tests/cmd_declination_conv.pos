( Easting, Northing, Altitude )
(328000.00, 5260000.00,  2500.00 ) test.0
(328029.36, 5260999.57,  2500.00 ) test.1
