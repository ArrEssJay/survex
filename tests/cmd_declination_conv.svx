; pos=yes warn=0
; Adapted from example posted by Mateusz Golicz to the mailing list
*begin test
*date 2015.08.01
*cs UTM33N
*cs out UTM33N
*fix 0 328000 5260000 2500
*declination auto 328000 5260000 2500
*data normal from to tape compass clino
0       1       1000   356.9275   0
*end test
