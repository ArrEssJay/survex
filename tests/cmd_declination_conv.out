
Removing trailing traverses...

Concatenating traverses...

Simplifying network...

Calculating network...

Calculating traverses...

Calculating trailing traverses...

Calculating statistics...

Survey contains 2 survey stations, joined by 1 leg.
There are 0 loops.
Total length of survey legs = 1000.00m (1000.00m adjusted)
Total plan length of survey legs = 1000.00m
Total vertical length of survey legs =    0.00m
Vertical range = 0.00m (from test.1 at 2500.00m to test.1 at 2500.00m)
North-South range = 999.57m (from test.1 at 5260999.57m to test.0 at 5260000.00m)
East-West range = 29.36m (from test.1 at 328029.36m to test.0 at 328000.00m)
   2 1-nodes.
